public/app/plugins/panel/geomap/layers/data/routeLayer.tsx

```diff
@@ -222,19 +247,35 @@ export const routeLayer: MapLayerRegistryItem<RouteConfig> = {
         .pipe(throttleTime(8))
         .subscribe({
           next: (event) => {
+            const mapExtents = map.getView().calculateExtent(map.getSize());
             const feature = source.getFeatures()[0];
             const frame: DataFrame = feature?.get('frame');
             const time: number = event.payload?.point?.time;
             if (frame && time) {
-              const timeField = frame.fields.find((f) => f.name === TIME_SERIES_TIME_FIELD_NAME);
+              const timeField = frame.fields.find((f) => f.type === FieldType.time);
```

Comment:
this will default to returning the first time field in the dataframe right? (though I'm guessing cases where there are multiple time fields is rare)

Reply:
Yeah, that's my understanding too.